### PR TITLE
Apptimize Root integration

### PR DESCRIPTION
**Pull request by f2prateek**

### CODE REVIEW

File: analytics-integrations/apptimize/src/main/java/com/segment/analytics/internal/integrations/ApptimizeIntegration.java [hunk 1]
```diff
@@ -17,12 +19,18 @@
...
+public class ApptimizeIntegration extends AbstractIntegration<Void>
+    implements OnExperimentRunListener {
   static final String APPTIMIZE_KEY = "Apptimize";
+  Analytics analytics;
 
   @Override public void initialize(Analytics analytics, ValueMap settings)
       throws IllegalStateException {
+    this.analytics = analytics;
     Apptimize.setup(analytics.getApplication(), settings.getString("appkey"));
+    if (settings.getBoolean("listen", false)) {
```
Comment (f2prateek) [inline analytics-integrations/apptimize/src/main/java/com/segment/analytics/internal/integrations/ApptimizeIntegration.java]: what will this setting be named?

**Comment by f2prateek** [comment]
cc @amillet89, refactored all the cruft to simplify roots. Let me know if this one looks good for Apptimize.

File: analytics-integrations/apptimize/src/main/java/com/segment/analytics/internal/integrations/ApptimizeIntegration.java (hunk 1 above)
Comment (amillet89) [inline analytics-integrations/apptimize/src/main/java/com/segment/analytics/internal/integrations/ApptimizeIntegration.java]: Right now we've labeled it as "Send experiment data to other tools"

**Comment by amillet89** [comment]
lgtm @f2prateek!

File: analytics-integrations/apptimize/src/main/java/com/segment/analytics/internal/integrations/ApptimizeIntegration.java (hunk 1 above)
Comment (f2prateek) [inline analytics-integrations/apptimize/src/main/java/com/segment/analytics/internal/integrations/ApptimizeIntegration.java]: And is `listen` the right key in the metadata for it?

**Comment by f2prateek** [comment]
Waiting on an Apptimize update that lets us test this end to end.

**Comment by mpoon** [comment]
@f2prateek 

On master, Segment is using Apptimize 2.6.11 which should be good to go for testing this change. Is there anything else you need from us?

**Comment by f2prateek** [comment]
@mpoon The Visual Aptimizer wasn't working on my device (Nexus 5, Android 5.1.1) so I couldn't setup an experiment and test this change.

**Comment by mpoon** [comment]
@f2prateek 
Ah ok, that makes sense as our SDK currently doesn't support visual experiments for Android 5.1.x or 5.0.2 (odd that this wasn't communicated on our UI). Support for those will be available later this month. Is it easy to test on a different Android version?

**Comment by f2prateek** [comment]
@mpoon We don't have an older device to test it on manually unfortunately, I tried using Genymotion emulators without any luck. 

@amillet89 what's the timeline on the roots launch? Would it be ok to wait for the Apptimize update until the end of the month?

**Pull request merged**